types/pdfmake/index.d.ts

```diff
@@ -145,18 +145,18 @@ declare module 'pdfmake/build/pdfmake' {
     }
 
     interface Table {
-        widths?: Array<(string | number)>;
-        heights?: Array<(string | number)> | TableRowFunction;
+        widths?: Array<(string | number | TableRowColFunction)>;
+        heights?: string | number | TableRowColFunction | Array<(string | number | TableRowColFunction)>;
         headerRows?: number;
         body: Content[][] | TableCell[][];
         layout?: string | TableLayoutFunctions;
     }
 
     interface Content {
-        style?: 'string';
+        style?: string;
```

Comment:
this, however, is still a small concern for me... technically as per the [style dictionaries](https://pdfmake.github.io/docs/document-definition-object/styling/#style-dictionaries) docs, this can also be an array of strings. so perhaps this should be adjusted to `string | string[]` first?

Reply:
with that said, having `style` as an array of strings *doesn't* actually throw any errors...

perhaps because of the `[additionalProperty: string]: any;` in `interface Content`?

Reply:
I guess `style` should be `string | string[]` because it is possible to define "functional" styles such as `font-bold` and `color-red` and then it would be useful to be able to use `style: ['font-bold', 'color-red']`.

To answer the second question, it should fallback to `[additionalProperty: string]: any;` and therefore allow the array.